docs/builtins/Operators/add.md

```diff
@@ -0,0 +1,56 @@
+## +
+The `+` function performs addition for numbers, concatenation for strings/lists, or merging for objects.
+
+### Basic syntax
+
+To perform addition for numbers, use the following syntax:
+
+`(+ x y)`
+
+To concatenate strings/lists or merge objects, use the following syntax:
+
+`(+ x y)`
+
+### Arguments
+
+Use the following arguments to specify the values for addition, concatenation, or merging using the `+` Pact function.
+
+| Argument | Type | Description |
+| --- | --- | --- |
+| `x` | `<a[integer,decimal,string,[<l>],object:<{o}>]>` | Specifies the first operand. |
+| `y` | `<a[integer,decimal,string,[<l>],object:<{o}>]>` | Specifies the second operand. |
+
+### Return value
+
+The `+` function returns the result of addition for numbers, concatenation for strings/lists, or merging for objects.
+
+### Examples
+
+The following examples demonstrate the usage of the `+` function within a Pact REPL. They perform addition, concatenation, or merging:
+
+```pact
+pact>(+ 1 2)
+3
+```
+
+```pact
+pact>(+ 5.0 0.5)
+5.5
+```
+
+```pact
+pact>(+ "every" "body")
+"everybody"
+```
+
+```pact
+pact>(+ [1 2] [3 4])
+[1 2 3 4]
+```
+
+```pact
+pact>(+ { "foo": 100 } { "foo": 1, "bar": 2 })
```

Comment:
I think it's worth mentioning explicitly somewhere whether merging objects is left-biased or right-biased when there are matching keys.